pandas/core/construction.py

```diff
@@ -773,6 +773,13 @@ def _try_cast(
             #  special case.
             return maybe_cast_to_datetime(arr, dtype)
             # TODO: copy?
+        from pandas import IntervalDtype
```

Comment:
this can be imported at the top (you need the extended form, e.g. `pandas.core.dtypes.dtype`

Reply:
Have import.